assets/radar-rules.js

```diff
@@ -1288,4 +1288,15 @@
             },
         ],
     },
+    'ustb.edu.cn': {
+        _name: '北京科技大学',
+        gs: [
+            {
+                title: '研究生院',
+                docs: 'https://docs.rsshub.app/university.html#bei-jing-ke-ji-da-xue',
+                source: '/:type',
+                target: '/ustb/news/:type',
+            },
+        ],
+    },
```

Comment:
Unnecessary addition.
This `radar-rules.js` is here for backwards compatibility.
Rules in /v2/ROUTE/radar.js from all routes will be aggregated into a new file. See https://github.com/DIYgod/RSSHub/issues/8269